red-hat-gallery/openshift-pipelines-gallery/base/adservice-pipelinerun.yaml

```diff
@@ -7,7 +7,7 @@ spec:
   - name: APP_NAME
     value: adservice
   - name: GIT_REPO
-    value: https://github.com/noseka1/red-hat-gallery
+    value: https://github.com/stratus-ss/red-hat-gallery
```

Comment:
Let's keep the noseka1 source repo until we find an official location (multiple occurrences below).